byterun/Makefile.nt

```diff
@@ -16,6 +16,7 @@
 include Makefile.common
 
 CFLAGS=-DOCAML_STDLIB_DIR='"$(LIBDIR)"' $(IFLEXDIR)
+DFLAGS=$(CFLAGS) -DDEBUG
```

Comment:
Humorously, only the msvc64 Makefile actually defined DEBUG, so the `ocamlrund.exe` built in the other ports was in fact just a standard runtime!
Defining `DFLAGS` is vaguely as done in `Makefile`